src/classes/BDI_Donations.cls

```diff
@@ -727,8 +719,7 @@ public class BDI_Donations {
             }
             if (dataImport.PaymentImported__c != null
                     || dataImport.DonationImportStatus__c == Label.bdiMatchedApplyNewPayment
-                    || (isElevatePaymentDI(dataImport) && 
-                        dataImport.PaymentImportStatus__c == null)) {
+                    || isElevatePaymentDI(dataImport)) {
```

Comment:
isElevatePaymentDI doesn't need to be at the beginning of the if-statement?

Reply:
@KietTruongSFDO Just simplified this since all Elevate Payments should be fine entering.